File: lambda/go/sample_handler.go
package main

import (
	"context"
	"encoding/json"
	"fmt"

	"github.com/aws/aws-lambda-go/events"
	"github.com/aws/aws-lambda-go/lambda"
)

// Handler function for Lambda
func handler(ctx context.Context, request events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
	// Print the event received
	eventJSON, _ := json.Marshal(request)
	fmt.Println("Received Event:", string(eventJSON))

	// Your Lambda logic goes here
	message := "Revcieved message: " + string(eventJSON)
	// Creating a response
	responseBody, err := json.Marshal(map[string]interface{}{
		"message": message,
	})
	if err != nil {
		return events.APIGatewayProxyResponse{StatusCode: 500}, err
	}

	return events.APIGatewayProxyResponse{
		StatusCode: 200,
		Headers:    map[string]string{"Content-Type": "application/json"},
		Body:       string(responseBody),
	}, nil
}

func main() {
	// Start the Lambda handler
	lambda.Start(handler)
}

File: lambda/python/sample_handler.py
import json
from typing import Any, Dict

from loguru import logger


def lambda_handler(event: Any, context: Dict):
    try:
        logger.info(f"Recieved Event Message : {json.dumps(event, indent=2)}")

        return {
            "status": "success",
            "msg": f"Received  message: {json.dumps(event)}",
            "status_code": "200",
        }
    except Exception as e:
        logger.error("Exception occurred", e)